packages/mui-lab/package.json

```diff
@@ -46,7 +46,7 @@
     "@mui/system": "^5.14.8",
     "@mui/types": "^7.2.4",
     "@mui/utils": "^5.14.8",
-    "@mui/x-tree-view": "6.0.0-alpha.1",
+    "@mui/x-tree-view": "6.0.0-alpha.2",
```

Comment:
@flaviendelangle if I remember correctly we said we don't want to update this dependency? If that's the case we need to update the renovate's config so that we don't accidently merge changes in the lab.

Reply:
Yes, I forgot to update it :+1:

Reply:
I fixed the CI checks, but didn't update the renovate config, because as per [the warning here ](https://github.com/mui/material-ui/blob/master/packages/mui-lab/src/TreeView/TreeView.tsx#L12) the component will be removed soon from Lab.